firmware/src/widgets/stockwidget/StockDataModel.cpp

```diff
@@ -139,3 +139,12 @@ StockDataModel &StockDataModel::setChangedStatus(bool changed) {
     m_changed = changed;
     return *this;
 }
+
+bool StockDataModel::isInitialized() {
+    return m_initialized;
+}
+
+StockDataModel &StockDataModel::setInitializationStatus(bool initialized) {
+    m_initialized = initialized;
+    return *this;
```

Comment:
Return value not needed/used.

Reply:
Yep, was just trying to follow the pattern